src/services/navigator/index.ts

```diff
@@ -58,7 +58,7 @@ export class Navigator {
   sendRequest = (action: NavAction, routeParams?: NavigationRouteParams) => {
     const navAction: NavAction = Helpers.getNavAction(action, routeParams);
 
-    const [navigation, routeId, params] = Helpers.buildRequest(
+    const [requestAction, navigation, routeId, params] = Helpers.buildRequest(
```

Comment:
there's a bit of tangling we can remove here:
- `navAction` is now only used as an entry param of `buildRequest`, so `Helpers.buildRequest` interface can be adjusted to receive `action` instead of `navAction`, and delegate to `Helpers.getNavAction` internally.
- instead of `requestAction`, we can preserve the name `navAction` which will help reduce the surface of the change